test/e2e/vsphere_volume_ops_strom.go

```diff
@@ -0,0 +1,188 @@
+			Expect(err).NotTo(HaveOccurred())
+			// Get new copy of the claim
+			claim, err := client.CoreV1().PersistentVolumeClaims(namespace).Get(claim.Name, metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+
+			// Get the bounded PV
+			persistentvolume, err := client.CoreV1().PersistentVolumes().Get(claim.Spec.VolumeName, metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			persistentvolumes = append(persistentvolumes, persistentvolume)
+		}
+
+		By("Creating pod to attach PV to the node")
+		podSpec := &v1.Pod{
+			TypeMeta: metav1.TypeMeta{
+				Kind:       "Pod",
+				APIVersion: "v1",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				GenerateName: "pod-30-volumes-",
+			},
+			Spec: v1.PodSpec{
+				Containers: []v1.Container{
+					{
+						Name:    "volume-tester",
+						Image:   "gcr.io/google_containers/busybox:1.24",
+						Command: []string{"/bin/sh"},
+						Args:    []string{"-c", "while true ; do sleep 2 ; done"},
+					},
+				},
+				RestartPolicy: v1.RestartPolicyNever,
+			},
+		}
+		var volumeMounts []v1.VolumeMount
+		var volumes []v1.Volume
+		for index, pvclaims := range pvclaims {
+			volumename := fmt.Sprintf("volume%v", index+1)
+			volumeMounts = append(volumeMounts, v1.VolumeMount{Name: volumename, MountPath: "/mnt/" + volumename})
+			volumes = append(volumes, v1.Volume{Name: volumename, VolumeSource: v1.VolumeSource{PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{ClaimName: pvclaims.Name, ReadOnly: false}}})
+		}
+		podSpec.Spec.Containers[0].VolumeMounts = volumeMounts
+		podSpec.Spec.Volumes = volumes
+
+		pod, err := client.CoreV1().Pods(namespace).Create(podSpec)
+		Expect(err).NotTo(HaveOccurred())
+
+		By("Waiting for pod to be running")
+		Expect(framework.WaitForPodNameRunningInNamespace(client, pod.Name, namespace)).To(Succeed())
+
+		// get fresh pod info
+		pod, err = client.CoreV1().Pods(namespace).Get(pod.Name, metav1.GetOptions{})
+		Expect(err).NotTo(HaveOccurred())
+
+		nodeName := pod.Spec.NodeName
+		vsp, err := vsphere.GetVSphere()
+		Expect(err).NotTo(HaveOccurred())
+
+		By("Verify all volumes are accessible and available in the pod")
+		for index, pv := range persistentvolumes {
+			// Verify disks are attached to the node
+			isAttached, err := verifyVSphereDiskAttached(vsp, pv.Spec.VsphereVolume.VolumePath, k8stype.NodeName(nodeName))
+			Expect(err).NotTo(HaveOccurred())
+			Expect(isAttached).To(BeTrue(), fmt.Sprintf("disk %v is not attached with the node", pv.Spec.VsphereVolume.VolumePath))
+			// Verify Volumes are accessible
+			filePath := "/mnt/" + fmt.Sprintf("volume%v", index+1) + "/emptyFile.txt"
+			_, err = framework.LookForStringInPodExec(namespace, pod.Name, []string{"/bin/touch", filePath}, "", time.Minute)
+			Expect(err).NotTo(HaveOccurred())
+		}
+
+		By("Deleting pod")
+		err = client.CoreV1().Pods(namespace).Delete(pod.Name, nil)
```

Comment:
can use `framework.DeletePodWithWait` to be sure that the pod is really deleted.